packages/repository-json-schema/src/build-schema.ts

```diff
@@ -36,68 +36,95 @@ export function getJsonSchema(ctor: Function): JSONSchema {
  * Gets the wrapper function of primitives string, number, and boolean
  * @param type Name of type
  */
-export function stringTypeToWrapper(type: string): Function {
-  type = type.toLowerCase();
-  let wrapper;
-  switch (type) {
-    case 'number': {
-      wrapper = Number;
-      break;
-    }
-    case 'string': {
-      wrapper = String;
-      break;
-    }
-    case 'boolean': {
-      wrapper = Boolean;
-      break;
-    }
-    default: {
-      throw new Error('Unsupported type: ' + type);
+export function stringTypeToWrapper(type: string | Function): Function {
+  if (typeof type === 'function') {
+    return type;
+  } else {
```

Comment:
Please remove this `else` block to reduce indentation level and keep the changeset smaller.

```ts
if (typeof type === 'function') {
  return type;
}

type = type.toLowerCase();
let wrapper;
switch(type) {
// ...
```